tests/backpressure_test/wscript

```diff
@@ -1,220 +1,14 @@
-import xwaf.utils.configs as config_utils
```

Comment:
It looks like this test has had configs removed as part of the infrastructure update - we should check if that's true/desired.